### ISSUE TITLE
[Bug]: Right-click drop-down menu of some chat messages hidden behind other messages

### ISSUE THREAD

**Issue by chaoticnote** [mclemente/fvtt-simple-chat-tabs#1]
**Foundry Version:** Version 12 Build 328
**Module Version**: 1.0.0

So to be exact, this is a bug I found when it is enabled with "Karakara's Chat Enhancements". When "Simple Chat Tabs" module is enabled with Kara's module, some right-click drop-down menus of some chat messages are hidden behind others as shown below.

![image](https://github.com/mclemente/fvtt-simple-chat-tabs/assets/28378016/be376558-8fc9-4786-9394-9f12759bb223)
_Bugged drop-down menu for editing options (Liljya)_
![image](https://github.com/mclemente/fvtt-simple-chat-tabs/assets/28378016/ad809668-0eac-497c-87a5-0a1aaf24d978)
_Bug replicated on player side in Chrome browser: Version 126.0.6478.127 (Official Build) (64-bit)_

![image](https://github.com/mclemente/fvtt-simple-chat-tabs/assets/28378016/9e5a314d-a8aa-4369-8648-a459dc947de0)
_Normal drop-down menu appearance example (Jirek)_

Now, I understand this may perhaps be a little much to ask, but if possible, could you find out a way to have it compatible with Karakara's chat enhancement module please? Or if you and Karakara (on Codeberg.org) could have a chat and figure out how to make both modules not conflict with this bug?

**Comment by mclemente** [comment]
I couldn't reproduce this error. Are you testing this with just both modules (and dependencies) enabled?
Also, which system are you using?

**Comment by chaoticnote** [comment]
Hiya. Admittedly I may have tested this my usual game world with all other modules installed, but at the time I distinctly remember turning either yours or Kara's module on and off for testing. As for the game system I was using, it's Pathfinder 2nd Edition. I was using an older version of it at the time, but I have since updated to PF2e version 6.1.2 and also kept other modules I use up to date.
As of now, I cannot replicate this bug even when going back to those same messages I've shown in the example. It may have been resolved with an update. As for what update, I'm not sure. Could be Foundry, the PF2e, or any of the modules I'm using. Sorry for not being able to provide any more useful information beyond that.
